src/core/nis1/Nis1Deadline.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ChronoUnit, Duration, Instant, LocalDateTime, ZoneId } from '@js-joda/core';
+/**
+ * The deadline of the transaction. The deadline is given as the number of seconds elapsed since the creation of the nemesis block.
+ * If a transaction does not get included in a block before the deadline is reached, it is deleted.
+ */
+export class Nis1Deadline {
+    /**
+     * Deadline value (without Nemesis epoch adjustment)
+     */
+    public adjustedValue: number;
```

Comment:
is this ms or s?

Reply:
hmm, just noticed now Nis1 is using seconds and Symbol is using milliseconds, will need to refactor the Deadline class I think, in a different PR perhaps